R/fonts.R

```diff
@@ -0,0 +1,72 @@
+#' Check font aliases for missing families and faces
+#' 
+#' @param aliases A list of families of faces. Typically created with
+#'   \code{fontquiver::font_families()}.
+#' @export
+check_aliases <- function(aliases) {
```

Comment:
In that case, add that motivation to the description, and show a couple of examples of use.